### PR TITLE
build: update MDC and MDC operator

### PR DESCRIPTION
**Pull request by craicoverflow**
Updated version of MDC.

## Verification

Verify that the mobile-developer-console tagged image exists.

`docker pull quay.io/aerogear/mobile-developer-console:1.1.9`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,81 +7,87 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
-## [0.2.2] - 2019-08-15
```
Comment (aliok) [inline CHANGELOG.md:10]: @craicoverflow 

We use the format defined here: https://keepachangelog.com/en/1.0.0/

I see you noticed some problems in the file and fixed them, but there's just one minor problem. The versions should start with 2 `#` chars.

It should be like
```
# Changelog
...

## [Unreleased]
 <Nothing here>

## [0.2.3] - 2019-08-23
### Changed
- Updaed version of Mobile Developer Console

## [0.2.2] - 2019-08-15
...
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,81 +7,87 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+### [0.2.3] - 2019-08-23
```
Comment (camilamacedo86) [inline CHANGELOG.md]: If you add # one here means that is is a subtopic of unreleased which is not the case. 
IMHO it should came back for the default standard.

**Review by aliok — APPROVED** [review]
Looks good, no need to verify.

...other than the minor thing I noted about the changelog.

File: version/version.go [hunk 3]
```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "0.2.2"
+	Version = "0.2.3"
 )
```
Comment (camilamacedo86) [inline version/version.go:5]: It is missing update the tags in the SOP files.

**Review by camilamacedo86 — CHANGES_REQUESTED** [review]
It is missing a few steps regards the SOP. ( see in the readme ).
Also, IMHO we should not perform the changes made in te CHANGELOG file and it should be kept as before in order to follow the standard.

File: CHANGELOG.md (hunk 1 above)
Comment (craicoverflow) [reply CHANGELOG.md:10]: Ah thanks, I will fix that.

File: version/version.go (hunk 3 above)
Comment (craicoverflow) [reply version/version.go:5]: > It is missing a few steps regards the SOP. ( see in the readme ).
> Also, IMHO we should not perform the changes made in te CHANGELOG file and it should be kept as before in order to follow the standard.

I'll do that now. Sure, I wasn't aware there was a standard for this (but how we had it was a little bit wrong).

**Comment by craicoverflow** [comment]
> It is missing a few steps regards the SOP. ( see in the readme ).
> Also, IMHO we should not perform the changes made in te CHANGELOG file and it should be kept as before in order to follow the standard.

Thanks. I was not aware of the SOP steps. Can you check again? I have also reverted the Changelog.

File: CHANGELOG.md (hunk 1 above)
Comment (camilamacedo86) [reply CHANGELOG.md:10]: 👍

File: version/version.go (hunk 3 above)
Comment (camilamacedo86) [reply version/version.go]: Hi @craicoverflow ,

Se that the roles still not pointing to the new tag in the following files.
- https://github.com/aerogear/mobile-developer-console-operator/blob/master/deploy/monitor/prometheus_rule.yaml#L21
- https://github.com/aerogear/mobile-developer-console-operator/blob/master/deploy/monitor/mdc_prometheus_rule.yaml

Comment (camilamacedo86) [reply version/version.go:5]: No problem that all.

**Review by camilamacedo86 — APPROVED** [review]
Hi @craicoverflow,

I am approving but please ....see that the roles still not pointing to the new tag in the following files. Just it is missing here. Could you please update them before merge? 

- https://github.com/aerogear/mobile-developer-console-operator/blob/master/deploy/monitor/prometheus_rule.yaml#L21
- https://github.com/aerogear/mobile-developer-console-operator/blob/master/deploy/monitor/mdc_prometheus_rule.yaml

**Pull request merged**